File: HADObjcExamples/Banner300x250.m
//
//  Banner300x250.m
//  HADObjCExample
//
//  Created by Mihael Isaev on 27/06/16.
//  Copyright © 2016 Mihael Isaev. All rights reserved.
//

#import "Banner300x250.h"

@interface Banner300x250 () <HADBannerViewDelegate>
@property (weak, nonatomic) IBOutlet HADBannerView *bannerView;
@end

@implementation Banner300x250

-(void)viewDidLoad {
    [super viewDidLoad];
    [self.bannerView loadAdWithPlacementId:@"5b3QbMRQ" bannerSize:HADBannerSizeBlock300x250 delegate:self];
}

#pragma mark - HADBannerViewDelegate

-(void)HADViewDidLoadWithView:(HADBannerView *)view {
    NSLog(@"HADViewDidLoad");
}

-(void)HADViewWithView:(HADBannerView *)view didFailWithError:(NSError *)error {
    NSLog(@"HADViewDidFai:l %@", error);
}

-(void)HADViewDidClickWithView:(HADBannerView *)view {
    NSLog(@"HADViewDidClick");
}

@end

File: HADSwiftExamples/Banner300x250.swift
//
//  Banner300x250.swift
//  HADFramework
//
//  Created by Mihael Isaev on 22/06/16.
//  Copyright © 2016 Mihael Isaev. All rights reserved.
//

import UIKit
import HADFramework

class Banner300x250: UIViewController, HADBannerViewDelegate {
    @IBOutlet weak var bannerView: HADBannerView!
    
    override func viewDidLoad() {
        super.viewDidLoad()
        bannerView.loadAd(placementId: "5b3QbMRQ", bannerSize: .block300x250, delegate: self)
    }
    
    func HADViewDidLoad(view: HADBannerView) {
        print("AD LOADED")
    }
    
    func HADViewDidClick(view: HADBannerView) {
        print("CLICKED AD")
    }
    
    func HADView(view: HADBannerView, didFailWithError error: NSError?) {
        print("ERROR: \(error?.localizedDescription)")
    }
}
